portal/server/helpers/pageSpecificValidationErrors.test.js

```diff
@@ -250,6 +250,58 @@ describe('page specific validation errors', () => {
       expect(result).toEqual(expected);
     });
 
```

Comment:
Can we add tests around the fix please

Reply:
Added test to check if field values are null